packages/lib/src/core/Context/CoreContext.ts

```diff
@@ -1,5 +1,14 @@
 import { createContext } from 'preact';
 import { CommonPropsTypes } from './CoreProvider';
 import Language from '../../language/Language';
+import { Resources } from './Resources';
 
-export const CoreContext = createContext({ i18n: new Language(), loadingContext: '', commonProps: {} as CommonPropsTypes });
+export const CoreContext = createContext({
+    i18n: new Language(),
+    loadingContext: '',
+    commonProps: {} as CommonPropsTypes,
+    resources: new Resources()
```

Comment:
If `Resources` has already been initialised as a module in `core.ts` why are we initialising a new one here?

Reply:
Same as for Language, it's no necessary but helps some of the component tests pass, since we don't go through the Core when we test them.